README.md

```diff
@@ -18,3 +18,90 @@ yarn run build
 ```
 
 The output is written to `Filters/filter.txt`.
+
+###  DNS filter structure
+###  Ad servers
+#### These filter lists block domains that are used to provide and show advertisements on websites
+
+* [AdGuard Base filter](https://raw.githubusercontent.com/AdguardTeam/AdguardFilters/master/BaseFilter/sections/adservers.txt)
+
+* [AdGuard Base filter first-party servers](https://raw.githubusercontent.com/AdguardTeam/AdguardFilters/master/BaseFilter/sections/adservers_firstparty.txt)
+    
+* [AdGuard Base filter foreign servers](https://raw.githubusercontent.com/AdguardTeam/AdguardFilters/master/BaseFilter/sections/foreign.txt)
```

Comment:
What exactly does `foreign` mean here?

Reply:
foreign includes domains of websites we have no language-specific filter for yet